tests/trio112.py

```diff
@@ -0,0 +1,35 @@
+import trio
+
+with trio.open_nursery() as n:  # error: 0
+    n.start(...)
+
+with trio.open_nursery() as n:  # error: 0
+    n.start_soon(...)
+
+with trio.open_nursery() as n:  # error: 0
+    n.start_soon(n=7)
+
+with trio.open_nursery() as n:  # error in theory
+    n.start_soon(lambda n: n + 1)
+
+# safe?
+with trio.open_nursery() as n, open("") as o:
+    n.start(...)
```

Comment:
Bit suspicious for #22-like reasons, but since violating this rule is more of a style problem than a correctness issue let's err on the side of less alarms.

Reply:
Thinking about this one more, is it actually that big of a style problem to have multiple `withitem`s in a single `with`? And if the open was placed in it's own `with` it'd clearly be a TRIO112 (or TRIO302/111).

I changed it to be an error, but trivial to change it back.